### DISCUSSION TITLE
第2节课笔记(18班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#97] [笔记]
请提交笔记链接
Please submit the link to the notes.

**Comment by yestolife** [comment]
[第二课笔记，跑通大模型领域的 hello world](http://t.csdnimg.cn/ZBpkX)

**Comment by DrZhang1208** [comment]
[第二节课笔记，轻松玩转书生·浦语大模型demo](https://blog.csdn.net/qq_42866033/article/details/135434142?spm=1001.2014.3001.5502)

**Comment by tackhwa** [comment]
[第2节课笔记，轻松玩转书生·浦语大模型趣味demo](https://nutritious-chill-7a9.notion.site/demo-556ece1db5554ee895e0ba030382c9fb?pvs=4)

**Comment by lengbaihang** [comment]
https://blog.csdn.net/lengbaihang/article/details/135438627?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135438627%22%2C%22source%22%3A%22lengbaihang%22%7D

**Comment by xwhclaire** [comment]
[Lecture 2 Notes 轻松玩转书生·浦语大模型趣味Demo](https://github.com/xwhclaire/StudyPackages/blob/main/LLM%20training/%23%20Lecture%202%20%E8%BD%BB%E6%9D%BE%E7%8E%A9%E8%BD%AC%E4%B9%A6%E7%94%9F%C2%B7%E6%B5%A6%E8%AF%ADLLM%E8%B6%A3%E5%91%B3Demo.md)

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/2

**Comment by ZMcc18** [comment]
https://www.yuque.com/zixul/avxnzn/cddt6u1mwgvqnrmg?singleDoc# 《lesson2 书生·浦语大模型Demo笔记》

**Comment by ggfengxuan** [comment]
[2.书生·浦语大模型趣味Demo笔记](https://qa5023amair.feishu.cn/wiki/ZZ6PwdXzZiXBZDkcb0kcwsr3nJD?from=from_copylink)

**Comment by JinFish** [comment]
[第二节课笔记+作业](https://blog.csdn.net/wildland/article/details/135442514?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135442514%22%2C%22source%22%3A%22wildland%22%7D)
**进阶作业在笔记中**

**Comment by tianxia0907** [comment]
[第二节课笔记+作业](https://github.com/tianxia0907/internlm-tutorial/blob/main/helloworld/hello_world.md)

**Comment by ZiHAO-LI-cmd** [comment]
[书生·浦语大模型实战营 Lesson 2](http://t.csdnimg.cn/ZhFzr)

**Reply by mattheliu** [reply]
佬，这边贴笔记链接哦 😳

**Comment by scorpionfree98** [comment]
http://t.csdnimg.cn/AGNTg

**Comment by D-JI-Feng** [comment]
第二节课笔记https://k0ilmnk4dz3.feishu.cn/wiki/FB7zwOMyJi5PFSkU6qxcB3J8nhd?from=from_copylink

**Comment by zxp22** [comment]
https://zhuanlan.zhihu.com/p/677150426

**Comment by 2001wjh** [comment]
https://www.yuque.com/wjh2001/rdvfen/ygqlm1vf5cuuwaav?singleDoc# 《轻松玩转书生·浦语大模型趣味Demo（笔记&作业）》

**Comment by Heitao5200** [comment]
https://gitee.com/heitao5200/my_blog/blob/master/docs/%E7%BB%84%E9%98%9F%E5%AD%A6%E4%B9%A0/%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%AE%9E%E6%88%98/TASK2.md
